EXILED/Exiled.Loader/Exiled.Loader.csproj

```diff
@@ -17,6 +17,7 @@
     </ItemGroup>
 
     <ItemGroup>
+        <PackageReference Include="Mono.Posix-4.5" Version="4.5.0" />
```

Comment:
Turn this into a target variable for consistency and future use and updatability.

Reply:
I don't think they are going to update it.
![imagen](https://github.com/user-attachments/assets/022bbf8c-2c3b-45b0-a6c7-0bae220eaa0d)

Reply:
> I don't think they are going to update it. ![imagen](https://private-user-images.githubusercontent.com/36207738/365029552-022bbf8c-2c3b-45b0-a6c7-0bae220eaa0d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU2MDE1MTgsIm5iZiI6MTcyNTYwMTIxOCwicGF0aCI6Ii8zNjIwNzczOC8zNjUwMjk1NTItMDIyYmJmOGMtMmMzYi00NWIwLWE2YzctMGJhZTIyMGVhYTBkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA2VDA1NDAxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFkY2VlNWNkODQxZThmOWU1YTdlOGI3YjNjYmY1NDNjOGEzMmQ4ZmNhMmU1ODc0ZWU0YmYyMWUxZmE2OTNkOTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oUmDVVoMutFO6PNa9kKU8P9Ce01YXP8V_yUpcmdmHqw)

oh lmao. All good then, I'm fine with this.